webapp/Connector/src/view/Learn.js

```diff
@@ -830,12 +830,22 @@ Ext.define('Connector.view.Learn', {
 
         if (dimension) {
             this.loadDataView(dimension, id, urlTab, params);
+            this.getHeader().showExportButton(dimension);
         }
         else {
             this.getHeader().on('selectdimension', this.loadDataView, this, {single: true});
         }
 
-        this.getHeader().selectTab(dimension ? dimension.uniqueName : undefined, id, dimension, params);
+        if (params && Object.keys(params).length > 0) {
```

Comment:
It seems the rewrite is to get rid of the updateFilters calls when params is not empty. Can we just add a param `skipFiltersUpdate` to the respective functions. I think that would be much cleaner.

`selectTab : function(dimUniqueName, id, dimension, params, skipFiltersUpdate)`
`filterStoreFromUrlParams: function(id, dimension, params, skipFiltersUpdate)`

Reply:
Maybe i missing something but the rewrite is to call updateFilters() when the params is not empty

Reply:
Yep, sorry I meant to get rid of the updateFilters call when params is empty. so skipFiltersUpdate = !params || Object.keys(params).length === 0

Reply:
Got it! 
yes, much cleaner, now updated